sky/shell/platform/ios/FlutterView.h

```diff
@@ -5,10 +5,14 @@
 #ifndef FLUTTER_FLUTTERVIEW_H_
 #define FLUTTER_FLUTTERVIEW_H_
 
+#include "sky/services/semantics/semantics.mojom.h"
+
 #include <UIKit/UIKit.h>
 
 @interface FlutterView : UIView
 
+- (instancetype)initWithSemantics:(semantics::SemanticsServer*)semantics;
```

Comment:
We already have the semantics server ready when we initialize the view, so it seemed cleanest.  Is there a reason to instantiate the accessibility bridge later?